runhouse/resources/hardware/on_demand_cluster.py

```diff
@@ -518,13 +519,14 @@ def ssh(self, node: str = None):
             if not sky_key.exists():
                 raise FileNotFoundError(f"Expected default sky key in path: {sky_key}")
 
+            runner = SkySSHRunner(
+                ip=node or self.address,
+                ssh_user=ssh_user,
+                port=self.ssh_port,
+                ssh_private_key=sky_key,
```

Comment:
just making sure we're not causing any issues by passing in a Path object and not a string?